### PR TITLE
chore(website): fix broken storybook links, update topbar examples

### PR DESCRIPTION
**Pull request by nkrantz**
**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [ ] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: .changeset/selfish-eels-switch.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@twilio-paste/icons': patch
+---
+
+[Icons] update support and product support icons
```
Comment (TheSisb) [inline .changeset/selfish-eels-switch.md:5]:
```suggestion
[Icons] update ProductSupport and Support icons
```

File: .changeset/orange-dodos-brake.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/topbar': patch
+'@twilio-paste/core': patch
+---
+
+[Topbar] fix column gap in topbar actions
```
Comment (TheSisb) [inline .changeset/orange-dodos-brake.md:6]:
```suggestion
[Topbar] style adjustment to the column-gap in TopbarActions
```

File: packages/paste-website/src/pages/components/topbar/index.mdx [hunk 3]
```diff
@@ -123,11 +359,45 @@ User-specific controls (aligned to the right) such as:
     ProductSegmentIcon,
     ProductFlexIcon,
     ProductEmailAPIIcon,
+    ChevronDownIcon,
     SearchIcon,
+    StatusMenu,
```
Comment (TheSisb) [inline packages/paste-website/src/pages/components/topbar/index.mdx:364]: praise: wow, that's one heck of a composition

File: packages/paste-website/src/component-examples/TopbarExamples.ts [hunk 4]
```diff
@@ -1,222 +1,887 @@
...
+  const onClick = (status) => {
+    setConnectivity(ConnectivityObject[status]);
+    menu.hide();
+  };
+  const productSwitcher = useProductSwitcherState();
+  const [product, setProduct] = React.useState('twilio');
+  const userDialogList = useUserDialogListState();
+
+  return (
+    <Box minWidth="size70">
```
Comment (TheSisb) [inline packages/paste-website/src/component-examples/TopbarExamples.ts:259]: thought: should we remove this Box or adjust the minWidth down? On my laptop at fullscreen my examples overflow. See the bottom example here:

image

File: packages/paste-website/src/pages/components/topbar/index.mdx (hunk 3 above)
Comment (nkrantz) [reply packages/paste-website/src/pages/components/topbar/index.mdx:364]: seriously

**Review by serifluous — APPROVED** [review]

**Review by SiTaggart — APPROVED** [review]

**Pull request merged**